src/pages/sighting/encounters/Encounters.jsx

```diff
@@ -352,7 +352,7 @@ export default function Encounters({
             'timeSpecificity',
             'locationId',
           ]);
-          await addEncounter(sightingId, copiedProperties);
+          await addEncounter(sightingId, copiedProperties); // Do we really want to proceed as usual if there's an error here?
```

Comment:
once we migrate to `useMutate` i think we can remove the refresh function below. in that implementation data should only be updated if the mutation is successful